parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputCommitter.java

```diff
@@ -49,29 +50,48 @@ public void commitJob(JobContext jobContext) throws IOException {
   }
 
   public static void writeMetaDataFile(Configuration configuration, Path outputPath) {
-    if (configuration.getBoolean(ParquetOutputFormat.ENABLE_JOB_SUMMARY, true)) {
+    JobSummaryLevel level = ParquetOutputFormat.getJobSummaryLevel(configuration);
+
+    if (level != JobSummaryLevel.NONE) {
```

Comment:
To avoid having too many nested blocks, maybe invert the condition and exit early?